src/domain/vocabulary/helper/dependencies.ts

```diff
@@ -0,0 +1,108 @@
+import * as E from 'fp-ts/Either'
+import * as S from 'fp-ts/string'
+import * as P from 'fp-ts/Predicate'
+import * as N from 'fp-ts/number'
+import * as ORD from 'fp-ts/Ord'
+import * as R from 'fp-ts/Record'
+import { pipe } from 'fp-ts/function'
+import type { Show } from 'fp-ts/lib/Show'
+import type { Deps } from '../command'
+import type { VocabularyPort } from '../port'
+
+export type Option = (deps: Deps) => E.Either<VocabularyDependenciesError, Deps>
```

Comment:
It's not a `Deps`, it's a transient `Deps`, because not all the properties are set during the build.

Reply:
Of course, that's a `Partial` object